service.datadog.yaml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
This file will conflict with the service definition for the agent defined [here](https://github.com/DataDog/k8s-datadog-agent-ops/blob/main/service.datadog.yaml#L3), and it is used to power the datadog-agent entry in the service catalog

I don't know if the behavior is that they will keep replacing each other, or if both will be merged in the backend, but just in case, is it possible to rename the dd-service to something like `datadog-agent-test` for now?

Reply:
renamed to `datadog-agent-nightly`